Sources/GRPCCodeGen/Internal/Renderer/TextBasedRenderer.swift

```diff
@@ -1134,8 +1134,9 @@ struct TextBasedRenderer: RendererProtocol {
   /// Renders the specified code block.
   func renderCodeBlock(_ description: CodeBlock) {
     if let comment = description.comment { renderComment(comment) }
-    let item = description.item
-    renderCodeBlockItem(item)
+    if let item = description.item {
```

Comment:
The previous style had a cleaner call site; the yet was the impetus to change?

Reply:
`description.item` is now `Optional`, so we need this change (or for `renderCodeBlockItem` to take an optional)